### ISSUE TITLE
Initial Submission

### ISSUE THREAD

**Issue by shehbashahab** [shehbashahab/ToDoApp#1]
My app is complete, please review. // cc @codepathreview and @codepath

**Comment by nesquena** [comment]
Shehba,

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer to learn all practical Android development and best practices as quickly as possible. We also do a code review for each submitted project.

The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo) and to **schedule a short 5-10 minute phone conversation** [here](https://www.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1bHR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU). Navigate to [today's date and onward from there](https://www.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1bHR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU) and choose a 15-minute slot. Let us know if none of those times work.

Once you select a slot, can you make sure to **include best number** to reach you at to the invite? Look forward to chatting soon!

**Comment by shehbashahab** [comment]
Hi Nathan,

I have a new commitment during our scheduled phone conversation time. Could
we reschedule to a different day ­ same time on Wednesday or Friday?

Thanks,
Shehba

From:  Nathan Esquenazi <notifications@github.com>
Reply-To:  shehbashahab/ToDoApp
<reply+003743c274618adc989ae0634e9054a0024843459cc1967b92cf0000000111b1d2079
2a169ce05770860@reply.github.com>
Date:  Sunday, July 5, 2015 at 10:20 PM
To:  shehbashahab/ToDoApp <ToDoApp@noreply.github.com>
Cc:  SHEHBA SHAHAB <shehbashahab@gmail.com>
Subject:  Re: [ToDoApp] Initial Submission (#1)


Shehba,

Looks good, this exercise is intended in part to give you an introduction to
the general rhythm of this course. The course is entirely project based with
an app being assigned each week and then due the following week. Each
project builds on the last to help each engineer to learn all practical
Android development and best practices as quickly as possible. We also do a
code review for each submitted project.

The next step is to continue working on extensions to your todo app
<http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-ext
ending-your-todo>  and to schedule a short 5-10 minute phone conversation
here 
<https://www.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1b
HR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU> . Navigate to today's date
and onward from there
<https://www.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1b
HR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU>  and choose a 15-minute
slot. Let us know if none of those times work.

Once you select a slot, can you make sure to include best number to reach
you at to the invite? Look forward to chatting soon!


Reply to this email directly or view it on GitHub
<https://github.com/shehbashahab/ToDoApp/issues/1#issuecomment-118723522> .

**Comment by nesquena** [comment]
Can't chat this week outside of that time slot and I'm out next week. Let's chat the week of the 20th. Can you do 11:15am? 



—
Nathan Esquenazi
CodePath, Co-founder
http://codepath.com

**Comment by shehbashahab** [comment]
Yes, 11:15 am any day during the week of the 20th should be fine.

**Comment by nesquena** [comment]
Invite sent. Chat with you then.

**Comment by shehbashahab** [comment]
Can we move the call to 11:30? I am in a meeting that may run over.


From:  Nathan Esquenazi <notifications@github.com>
Reply-To:  shehbashahab/ToDoApp
<reply+003743c2691a65544b1b1551dbcb826e791b58de279e1ff892cf0000000111b5ce269
2a169ce05770860@reply.github.com>
Date:  Wednesday, July 8, 2015 at 10:52 PM
To:  shehbashahab/ToDoApp <ToDoApp@noreply.github.com>
Cc:  SHEHBA SHAHAB <shehbashahab@gmail.com>
Subject:  Re: [ToDoApp] Initial Submission (#1)


Invite sent. Chat with you then.


Reply to this email directly or view it on GitHub
<https://github.com/shehbashahab/ToDoApp/issues/1#issuecomment-119828466> .

**Comment by shehbashahab** [comment]
Persisted the todo items into SQLite instead of a text file.

**Comment by shehbashahab** [comment]
Alert Dialog prompting for deletion confirmation on long press
